quantmsio/core/diann.py

```
@@ -96,88 +97,122 @@ def get_peptide_count(self, df):
 
-            # cal value and mod
-            mass_vector = report["peptidoform"].map(masses_map)
-            report["calculated_mz"] = (mass_vector + (PROTON_MASS_U * report["precursor_charge"].values)) / report[
-                "precursor_charge"
-            ].values
+            # Apply q-value threshold
+            report_filtered = report[report["qvalue"] < qvalue_threshold]
 
-            report["peptidoform"] = report["peptidoform"].map(modifications_map)
+            # Define columns for final dataframe
+            usecols = report_filtered.columns.tolist() + ["scan", "observed_mz"]
```

Ruff (0.8.2)

Local variable `usecols` is assigned to but never used

Remove assignment to unused variable `usecols`

(F841)